compiler/src/compiler_test.go

```diff
@@ -2,22 +2,19 @@ package compiler
 
 import (
 	"fmt"
+	"github.com/rs/zerolog"
+	"io/ioutil"
 	"os/exec"
 	"testing"
-	"github.com/rs/zerolog"
-	"os"
 )
 
-func initLogger() (zerolog.Logger){
+func initLogger() zerolog.Logger {
 	zerolog.TimeFieldFormat = zerolog.TimeFormatUnix
-	consoleWriter := zerolog.ConsoleWriter{Out: os.Stdout}
+	consoleWriter := zerolog.ConsoleWriter{Out: ioutil.Discard}
 	logger := zerolog.
-					New(consoleWriter).
-					With().
-					Timestamp().
-					Str("MaRT", "executor").
-					Logger().
-					Level(zerolog.DebugLevel)
+		New(consoleWriter).
+		With().
+		Logger()
```

Comment:
Can't you just simplify to `zerolog.New(consoleWriter)`?

Reply:
This is true